book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'rentable'
+
+class Book
+  attr_accessor :title, :author
+
```

Comment:
- [ ] Kindly define the `@rentals` list as `attr_accessor` as you can access it inside the initialize method Rental class.

Reply:
done